src/app/features/shows/pages/movie-list/movie-list.component.html

```diff
@@ -1 +1,9 @@
-<p>movie-list works!</p>
+<span class="mat-body-1">MaileHereko</span>
+<h1 class="mat-headline-1">Movies</h1>
+<span class="mat-subtitle-2"> {{ count }} items</span>
```

Comment:
```suggestion
<span class="mat-subtitle-2"> {{ movies.length }} items</span>
```
ko cần tạo biến count, vì em chỉ dùng 1 lần